example-notebooks/binary-classifier/usage_config.yml

```diff
@@ -6,6 +6,7 @@ data_usage:
   # Each use case must define a primary output and target
   # Output should be in the predictions table but target may be a display name of a windowed event
   primary_output: LGBM_score
```

Comment:
When is the data update (and alignment) going through? This column name goes away right?  Are we going to need a versioning strategy - or message on how to update the right pieces?

Post change, I think metadata will also need updating because the score thresholds don't demonstrate the same things (see heatmap) with new score distributions

Reply:
I have the local files for testing the data update, so we can post the data update 1st (if you are fine with the update) and then I can add the new config to the example notebook in this PR, else we can do it in a future PR

Reply:
Updated now that https://github.com/epic-open-source/seismometer-data/pull/4 is merged, now uses the defaults from the updated diabetes dataset.